worksheet_A/TerminalHacking.cpp

```diff
@@ -29,17 +48,37 @@ int main()
 	// Using a set for options guarantees that the elements are all different
 	while (options.size() < numberOfWords)
 	{
-		std::string word = words.getRandomWord();
+		string word = words.getRandomWord();
 		options.insert(word);
 	}
 
 	// Display all words
-	for each (std::string word in options)
+	for each (string word in options)
 	{
-		std::cout << word << std::endl;
+		cout << word << endl;
 	}
-
-	// TODO: implement the rest of the game
+	
+	// Game loop
+	while (playerLives > 0) 
+	{
+		cout << "Choose a word... ";
+		cin >> guessWord;
```

Comment:
The flowchart in the worksheet says that you should check that the guessed word is in the list of options -- your program doesn't do this.